debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -162,6 +162,18 @@
                                                    .withDependents(COLLECTION_LIST_NAME)
                                                    .withDescription("The databases for which changes are to be captured");
 
+        /**
+     * A comma-separated list of regular expressions that match the fully-qualified namespaces of collections to be monitored.
+     * Fully-qualified namespaces for collections are of the form {@code '<databaseName>.<collectionName>'}.
+     * May not be used with {@link #COLLECTION_BLACKLIST}.
+     */
+    public static final Field DATABASE_WHITELIST = Field.create("database.whitelist")
```

Comment:
It would be nice to have a `database.blacklist`, too, so to resemble the collection-level options.